2021-09-30 12:40:32,873 INFO    MainThread:45153 [wandb_setup.py:_flush():69] setting env: {}
2021-09-30 12:40:32,873 INFO    MainThread:45153 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-30 12:40:32,873 INFO    MainThread:45153 [wandb_init.py:_log_setup():342] Logging user logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-2-data-exploration-and-preparation/exercises/exercise_6/starter/wandb/run-20210930_124032-1cek4y68/logs/debug.log
2021-09-30 12:40:32,873 INFO    MainThread:45153 [wandb_init.py:_log_setup():343] Logging internal logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-2-data-exploration-and-preparation/exercises/exercise_6/starter/wandb/run-20210930_124032-1cek4y68/logs/debug-internal.log
2021-09-30 12:40:32,873 INFO    MainThread:45153 [wandb_init.py:init():375] calling init triggers
2021-09-30 12:40:32,873 INFO    MainThread:45153 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-30 12:40:32,874 INFO    MainThread:45153 [wandb_init.py:init():424] starting backend
2021-09-30 12:40:32,874 INFO    MainThread:45153 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-09-30 12:40:32,883 INFO    MainThread:45153 [backend.py:ensure_launched():135] starting backend process...
2021-09-30 12:40:32,892 INFO    MainThread:45153 [backend.py:ensure_launched():139] started backend process with pid: 45165
2021-09-30 12:40:32,893 INFO    MainThread:45153 [wandb_init.py:init():429] backend started and connected
2021-09-30 12:40:32,896 INFO    MainThread:45153 [wandb_init.py:init():477] updated telemetry
2021-09-30 12:40:32,896 INFO    MainThread:45153 [wandb_init.py:init():500] communicating current version
2021-09-30 12:40:33,379 INFO    MainThread:45153 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-30 12:40:33,379 INFO    MainThread:45153 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-30 12:40:33,627 INFO    MainThread:45153 [wandb_init.py:init():540] starting run threads in backend
2021-09-30 12:40:34,103 INFO    MainThread:45153 [wandb_run.py:_console_start():1601] atexit reg
2021-09-30 12:40:34,104 INFO    MainThread:45153 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-09-30 12:40:34,104 INFO    MainThread:45153 [wandb_run.py:_redirect():1480] Redirecting console.
2021-09-30 12:40:34,104 INFO    MainThread:45153 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-30 12:40:34,104 INFO    MainThread:45153 [wandb_init.py:init():565] run started, returning control to user process
2021-09-30 12:40:34,293 INFO    MainThread:45153 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-09-30 12:40:34,293 INFO    MainThread:45153 [wandb_run.py:_restore():1543] restore
2021-09-30 12:40:36,639 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1247
  total_bytes: 1247
}

2021-09-30 12:40:38,822 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1247
  total_bytes: 1247
}

2021-09-30 12:40:39,039 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1247
  total_bytes: 1247
}

2021-09-30 12:40:39,669 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1247
  total_bytes: 7799
}

2021-09-30 12:40:39,776 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1247
  total_bytes: 7799
}

2021-09-30 12:40:39,882 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1247
  total_bytes: 7799
}

2021-09-30 12:40:39,988 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7483
  total_bytes: 7799
}

2021-09-30 12:40:40,096 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7483
  total_bytes: 7799
}

2021-09-30 12:40:40,204 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7799
  total_bytes: 7799
}

2021-09-30 12:40:40,309 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7799
  total_bytes: 7799
}

2021-09-30 12:40:40,419 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7799
  total_bytes: 7799
}

2021-09-30 12:40:40,527 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7799
  total_bytes: 7799
}

2021-09-30 12:40:40,636 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7799
  total_bytes: 7799
}

2021-09-30 12:40:41,037 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7799
  total_bytes: 7799
}

2021-09-30 12:40:41,144 INFO    MainThread:45153 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7799
  total_bytes: 7799
}

2021-09-30 12:40:42,224 INFO    MainThread:45153 [wandb_run.py:_show_files():1915] logging synced files
